pycrostates/preprocessing/__init__.py

```diff
@@ -5,5 +5,6 @@
 
 from .extract_gfp_peaks import extract_gfp_peaks
 from .resample import resample
+from .spatial_filter import apply_spatial_filter
 
-__all__ = ("extract_gfp_peaks", "resample")
+__all__ = ("extract_gfp_peaks", "resample", "apply_spatial_filter")
```

Comment:
```suggestion
__all__ = ("apply_spatial_filter", "extract_gfp_peaks", "resample")
```

Just to keep them ordered alphabetically 😅